.buildkite/pipeline.yml

```diff
@@ -57,12 +57,13 @@ steps:
         agents:
           queue: "benchmark"
           cuda: "*"
+          gpu: "rtx4070"
         if: build.message !~ /\[skip benchmarks\]/
         matrix:
           setup:
             julia:
               - "1.9"
-        timeout_in_minutes: 30
+        timeout_in_minutes: 300
```

Comment:
That's... long.

Reply:
Ah yes, I bumped this when I got a timeout, but forgot to change it to a more reasonable number :sweat_smile: